card/src/main/java/com/adyen/checkout/card/CardComponent.kt

```diff
@@ -151,7 +160,8 @@ class CardComponent private constructor(
                 unencryptedCardBuilder.setNumber(stateOutputData.cardNumberState.value)
             }
             if (!cardDelegate.isCvcHidden()) {
-                unencryptedCardBuilder.setCvc(stateOutputData.securityCodeState.value)
+                val cvc = stateOutputData.securityCodeState.value
+                if (cvc.isNotEmpty()) unencryptedCardBuilder.setCvc(cvc)
```

Comment:
Added this check because `unencryptedCardBuilder` only supports null cvc or length 3 and 4